.github/workflows/python-min-deps.yml

```diff
@@ -136,8 +136,6 @@ jobs:
         run: uv pip install --editable lib --no-deps
       - name: Run Python Tests
         run: make pytest
-      - name: Run Integration Tests
-        run: make integration-tests
```

Comment:
Running these with min versions would require that our tests are compatible with all dependency versions ... which adds unnecessary overhead to our e2e tests.